features/shared/entering_parent_branch.feature

```diff
@@ -31,6 +31,7 @@ Feature: Entering a parent branch name when prompted
     When I run `git sync` and enter "2" and "1"
     Then I see "Please specify the parent branch of feature-2"
     And I see "Please specify the parent branch of feature-1"
+    And I only see the branch list once
```

Comment:
We could use regex placeholders if we don't want to redundandly specify the header message. Something like

```cucumber
Then I see
  """
  <%= branch configuration header %>

  Please specify the parent branch of feature-2
  Please specify the parent branch of feature-1
  """
```

Then we only specify the header once in a spec dedicated to that, and the other specs just verify the presence of it.